packages/appstore/src/constants/tour-steps-config-new.tsx

```diff
@@ -55,8 +55,10 @@ export const getTourStepConfig = (): Step[] => [
                 <div className='toggle-account-type__divider' />
             </React.Fragment>
         ),
-        content: <Text as='p'>{localize('Click here if you ever need to repeat this tour.')}</Text>,
-        target: '.trading-hub-header__tradinghub--onboarding--logo',
+        content: (
```

Comment:
Why are you replacing `localize` with `Localize` component only for 2 of the strings in this file? is it really necessary?

Reply:
these 2 were visible in the PR changes files, so thought someones gonna ask to change it to Localize during review. so did this.
will do it through out the file 👍🏼 😄